### PR TITLE
job-detail

### PR DESCRIPTION
**Pull request by Rue-ops6**

### CODE REVIEW

File: resources/views/includes/detail.blade.php [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+                        </div>
+                        <div class="bg-light rounded p-5 wow slideInUp" data-wow-delay="0.1s">
+                            <h4 class="mb-4">Company Detail</h4>
+                            <p class="m-0">Ipsum dolor ipsum accusam stet et et diam dolores, sed rebum sadipscing elitr vero dolores. Lorem dolore elitr justo et no gubergren sadipscing, ipsum et takimata aliquyam et rebum est ipsum lorem diam. Et lorem magna eirmod est et et sanctus et, kasd clita labore.</p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <!-- Job Detail End -->
```
Comment (Basma-Abdallah) [inline resources/views/includes/detail.blade.php:87]: good job ya roro

File: resources/views/includes/header.blade.php [hunk 2]
```diff
@@ -4,11 +4,11 @@
                 <h1 class="display-3 text-white mb-3 animated slideInDown">About Us</h1>
                 <nav aria-label="breadcrumb">
                     <ol class="breadcrumb text-uppercase">
-                        <li class="breadcrumb-item"><a href="#">Home</a></li>
+                        <li class="breadcrumb-item"><a href="{{route('index')}}">Home</a></li>
```
Comment (Basma-Abdallah) [inline resources/views/includes/header.blade.php:7]: great , we need to fix all others links too

File: resources/views/layouts/main.blade.php [hunk 3]
```diff
@@ -9,9 +9,8 @@
 @include('includes.spinner')
 @include('includes.navbar')
 
-
-
 @yield('content')
+@yield('detail')
```
Comment (Basma-Abdallah) [inline resources/views/layouts/main.blade.php:13]: why did you add it to the template  I didn't get it ?

File: resources/views/includes/detail.blade.php (hunk 1 above)
Comment (Rue-ops6) [reply resources/views/includes/detail.blade.php:87]: tnx 7bibty

File: resources/views/layouts/main.blade.php (hunk 3 above)
Comment (Rue-ops6) [reply resources/views/layouts/main.blade.php:13]: I gave it a new name for a section to open to the detail section as otherwise it will open up the about section

**Comment by Nora-Lootfy** [comment]
it is a good practice to make it clear, what you are working on in pull request description

File: resources/views/layouts/main.blade.php (hunk 3 above)
Comment (Nora-Lootfy) [reply resources/views/layouts/main.blade.php:13]: I don't think it is a good practice to add two yields below each other, you add a yield if there is a code in between

not a good idea
```php
@yield('section1')
@yield('section2')
``` 

use it when
```php
@yield('section1')
{{-- some code in between --}}
@yield('section2')
```

File: resources/views/job-detail.blade.php [hunk 4]
```diff
@@ -1,264 +1,10 @@
...
-    <link href="https://fonts.googleapis.com/css2?family=Heebo:wght@400;500;600&family=Inter:wght@700;800&display=swap" rel="stylesheet">
-    
-    <!-- Icon Font Stylesheet -->
-    <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.10.0/css/all.min.css" rel="stylesheet">
-    <link href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.4.1/font/bootstrap-icons.css" rel="stylesheet">
 
-    <!-- Libraries Stylesheet -->
-    <link href="lib/animate/animate.min.css" rel="stylesheet">
-    <link href="lib/owlcarousel/assets/owl.carousel.min.css" rel="stylesheet">
+@section('detail')
```
Comment (Nora-Lootfy) [inline resources/views/job-detail.blade.php:6]: you can use `content` no need for extra section

**Review by Nora-Lootfy — CHANGES_REQUESTED** [review]
I have resolved a conflict for you before you change your code make sure to do this command in the terminal 

just remove the extra yield as no need for it

```console
git pull origin Rowan
```

so you can take the updates i have pushed to your branch

File: resources/views/layouts/main.blade.php (hunk 3 above)
Comment (Rue-ops6) [reply resources/views/layouts/main.blade.php:13]: ok doc

File: resources/views/job-detail.blade.php (hunk 4 above)
Comment (Rue-ops6) [reply resources/views/job-detail.blade.php]: done
